scripts/common/preflights.sh

```diff
@@ -899,3 +900,20 @@ function bail_if_kurl_version_is_lower_than_previous_config() {
         log "and the current kURL version used is $KURL_VERSION"
     fi
 }
+
+# bail if rook/openebs prereqs not met for minimumNodeCount param for automated storage scaling
+function bail_if_automated_storage_scaling_prereqs_not_met() {
+    if [ -n "ROOK_MINIMUM_NODE_COUNT"]; then
+        semverCompare "$(echo "$ROOK_VERSION" | sed 's/v//g')" "$(echo "1.11.5" | sed 's/v//g')"
```

Comment:
We need to settle on what is the minimum rook version to trigger automated storage scaling.

Reply:
1.11.x IMO